apps/src/netsim/NetSimPacketEditor.js

```diff
@@ -224,6 +233,14 @@ NetSimPacketEditor.prototype.getRoot = function () {
   return this.rootDiv_;
 };
 
+/**
+ * Returns the first visible message box, so that we can focus() on it
+ * @returns {jQuery}
+ */
+NetSimPacketEditor.prototype.getFirstVisibleMessage = function () {
```

Comment:
Nit: At first read, it wasn't clear to me from the name of this method that it's retrieving a reference to a a `textarea` element, rather than (say) its contents.  Maybe `getFirstVisibleMessageBox` would be a little more obvious?